### PR TITLE
Test Simba's JDBC driver's PreparedStatement 

### PR DESCRIPTION
**Pull request by brian-rickman**
Only 6738e8b and later commits are unique to this branch.  The others are from simba-jdbc-tests-2, which has an outstanding PR #5586 on prestodb:master.

I'd like to get feedback from TD engineers before posting another PR to prestodb:master.

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+import static java.sql.JDBCType.SMALLINT;
+import static java.sql.JDBCType.TIMESTAMP;
+import static java.sql.JDBCType.TINYINT;
+import static java.sql.JDBCType.VARBINARY;
+import static java.sql.JDBCType.VARCHAR;
+
+public class PreparedStatements
+        extends ProductTest
+{
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(JdbcTests.class);
```
Comment (rschlussel-zz) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java:67]: the rest of the Presto code uses io.airlift.log.Logger you should do the same here (and import it)

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+            implements RequirementsProvider
+    {
+        @Override
+        public Requirement getRequirements(Configuration configuration)
+        {
+            return mutableTable(ALL_HIVE_SIMPLE_TYPES_TEXTFILE, TABLE_NAME_MUTABLE, CREATED);
+        }
+    }
+
+    private Connection connection;
```
Comment (rschlussel-zz) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: this should be before the static classes

**Comment by rschlussel-zz** [comment]
Prepared statements tests look good, but note that the parameters stuff hasn't been merged to master yet.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java (hunk 2 above)
Comment (brian-rickman) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: I'll move it

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java (hunk 1 above)
Comment (brian-rickman) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java:67]: Tempto uses slf4j, and this code is more closely associated with Tempto than Presto, so I think this is OK.  Also, I used the same logging mechanism in #5586.  It was reviewed by Electrum and Grzegorz, and neither objected.

**Comment by brian-rickman** [comment]
This PR has been replaced by #385

**Pull request closed without merging**